### DISCUSSION TITLE
[A/L] (2021/22) Relazioni - Esercizio 6

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#278] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152318797-d5bbd06f-32b4-4acb-96e8-92d8467cc3f5.png)

**Comment by GattoBrizzolato** [comment]
1. V
2. F    poniamo il caso di relazioni di ordine inverse:  con a, b E(A) e 'a' DIVERSO DA 'b' otteniamo da una parte aRb mentre dall'altra bRa e questo contraddice la proprietà d'ordine 
3. F    poniamo il caso che in R ci sia (a, b) e (b, c) e (a, c) di conseguenza. Nel mentre in S poniamo solo (a, c). Chiaramente sottraendo (a, c) ad R verrebbe meno la proprietà transitiva

**Comment by LuginiAndrea** [comment]
La prima è vera, in quanto se:
riflessività: $aRa \wedge aSa \Longrightarrow aR^\*a$
anti-simmetria $aRb \wedge bRa \Longrightarrow a = b$, idem con $S$, ne consegue che vale anche per $R^*$
transitività: $aRb \wedge bRc \Longrightarrow aRc$, idem con $S$, ne consegue che vale anche per $R^\*$

La seconda:
riflessività: come prima
anti-simmetria: $aRb \vee aSb$: ipotizziamo $R = aRb \Longleftrightarrow a > b$ e $S= aSb \Longleftrightarrow a < b$. Prendiamo la coppia $a = 100, b = 11$: Vale $aR^*b$ in quanto $aRb$, ma vale anche $bR^*a$ in quanto $bSa$, ma $b \neq a$. Non è anti-simmetrica, quindi non è un ordine 

La terza: Come risposto sopra da @GattoBrizzolato

**Comment by Exyss** [comment]
@LuginiAndrea attenzione, nella dimostrazione dell'antisimmetria della seconda relazione hai scelto due relazioni che **non sono ordini parziali** poiché $a \nless a$ e $a \ngtr a$, dunque non vale la riflessività (scegliendo $\leq$ e $\geq$ la dimostrazione risulta essere corretta)

-------


## $R \cap S$

Per comodità ridenominiamo $R \cap S$ come $\prec$ ed affermiamo che $x \prec y \iff (x,y) \in R \land (x,y) \in S$

#### Riflessività :heavy_check_mark: 
$a \in A \implies (a,a) \in R \land (a,a) \in S \implies a \prec a$

#### Antisimmetria :heavy_check_mark: 
$a \prec b \land b \prec a \implies (a,b), (b,a) \in R \land (a,b), (b,a) \in S \implies a = b$

(poiché per simmetria $(a,b),(b,a) \in R \implies a= b$, analogamente per S)

#### Transitività :heavy_check_mark: 
$a \prec b \land b \prec c \implies (a,b), (b,c) \in R \land (a,b),(b,a) \in S \implies (a,c) \in R \land (a,c) \in S \implies a \prec c$

------

## $R \cup S$

Per comodità ridenominiamo $R \cup S$ come $\prec$ ed affermiamo che $x \prec y \iff (x,y) \in R \lor (x,y) \in S$

#### Riflessività :heavy_check_mark: 

$a \in A \implies (a,a) \in R \land (a,a) \in S \implies a \prec a$

#### Antisimmetria :no_entry_sign: 

$a \prec b \land b \prec a \implies ((a,b) \in R \lor (a,b) \in S) \land ((b,a) \in R \lor (b,a) \in S))$

Supponiamo quindi il caso in cui $(a,b) \in R, (a,b) \notin S, (b,a) \notin R, (b.a) \in S$.

Poiché
- $(a,b) \in R, (b.a) \notin R \centernot\implies a = b$
- $(a,b) \notin S, (b.a) \in S \centernot\implies a = b$

non abbiamo modo di accertarci che $a = b$, dunque l'antisimmetria non persiste in tutti i casi

------

## $R - S$

Per comodità ridenominiamo $R -S$ come $\prec$ ed affermiamo che $x \prec y \iff (x,y) \in R \land (x,y) \notin S$

#### Riflessività :no_entry_sign: 

$a \in A \implies (a,a) \in R \land (a,a) \in S \implies a \not\prec a$
